<launch>

	<!-- Parameters setting -->
	<param name="bot_client_key" type="str" value="5f2c3014e1c6460d9714984ff5405c9a"/>
	<param name="yandex_voice_key" type="str" value="49d9bb75-7419-45cc-9988-76052abc6c44"/>
	<param name="pyaudio" type="str" value={"format": 8, "channels": 1, "rate": 16000, "frames_per_buffer": 1024}/>
	<param name="algorithmia_api_key" type="str" value="simAkHd6NFelROawvgBT9ouDB6C1"/>
	
	
	<!-- Nodes running -->
	
	<node name="audio_capture" pkg="coffebot" type="audio_capture.py" output="screen"/>
	<node name="audio_min_volume_detector" pkg="coffebot" type="audio_min_volume_detector.py" output="screen"/>
	<node name="audio_player" pkg="coffebot" type="audio_player.py" output="screen"/>
	<node name="core_decision_manager" pkg="coffebot" type="core_decision_manager.py" output="screen"/>
	<node name="core_motion_manager" pkg="coffebot" type="core_motion_manager.py" output="screen"/>
	<node name="dialog_bot_manager" pkg="coffebot" type="dialog_bot_manager.py" output="screen"/>
	<node name="speech_recognition" pkg="coffebot" type="speech_recognition.py" output="screen"/>
	<node name="speech_synthesizer" pkg="coffebot" type="speech_synthesizer.py" output="screen"/>
	<node name="audio_player" pkg="coffebot" type=".py" output="screen"/>
	<node name="vision_camera_capture" pkg="coffebot" type="vision_camera_capture.py"/>
	<node name="vision_face_tracking" pkg="coffebot" type="vision_face_tracking.py" output="screen"/>
	<node name="vision_face_recognition" pkg="coffebot" type="vision_face_recognition.py" output="screen"/>
	<node name="vision_photo_capture" pkg="coffebot" type="vision_photo_capture.py" output="screen"/>
	<node name="vision_video_capture" pkg="coffebot" type="vision_video_capture.py" output="screen"/>
	<node name="motion_body_controller" pkg="coffebot" type="motion_body_controller.py" output="screen"/>
	<node name="motion_eyebrows_controller" pkg="coffebot" type="motion_eyebrows_controller.py" output="screen"/>
	<node name="motion_eyes_controller" pkg="coffebot" type="motion_eyes_controller.py" output="screen"/>
	<node name="motion_head_controller" pkg="coffebot" type="motion_head_controller.py" output="screen"/>
	
</launch>
